nexus/src/app/sagas/region_snapshot_replacement_step.rs

```diff
@@ -361,20 +364,40 @@ async fn rsrss_replace_snapshot_in_volume(
         .await
         .map_err(ActionError::action_failed)?;
 
-    Ok(())
+    info!(
```

Comment:
i wonder whether this should be info or debug?

it might also be nice to include the volume ID and stuff here, unless that's in the saga context?

Reply:
changed to debug in d2135f21b, and yeah, same answer - it's even more so in the context for this saga because the entire replace params is.

Reply:
:+1: